foreman/data_refinery_foreman/foreman/management/commands/create_quantpendia.py

```diff
@@ -72,10 +91,10 @@ def handle(self, *args, **options):
         logger.debug('Generating quantpendia for organisms', organisms=all_organisms)
 
         for organism in all_organisms:
-            # only generate the quantpendia for organisms that have some samples
-            # with quant.sf files.
+            # only generate the quantpendia for organisms that at least one
+            # RNA-SEQ processed sample
```

Comment:
What's the reason for this change? This is significantly altering the existing behavior.

Reply:
I thought they were equivalent and I wanted some alternative to avoid doing `.filter(results__computedfile__filename='quant.sf')` which is very expensive for the DB.

This will exclude any organisms that have `quant.sf` files but don't have any processed RNA-SEQ samples. Currently, no organism in our DB meets this condition. but if we were to have one, do we want a quantpendia for them? Note that people can download `quant.sf` files through the smasher as well.

```py
>>> Organism.objects.filter(qn_target__isnull=False).exclude(sample__is_processed=True, sample__technology='RNA-SEQ').filter(sample__results__computedfile__filename='quant.sf').exists()
False
```

Reply:
Hmm, not too sure, but I guess my gut is "if we can, why not?"

Reply:
Good point reverted the condition.